avro-fastserde/src/main/java/com/linkedin/avro/api/PrimitiveFloatList.java

```diff
@@ -0,0 +1,11 @@
+package com.linkedin.avro.api;
+
+import java.util.List;
+
+/**
+ * A {@link List} implementation with additional functions to prevent boxing.
+ */
+public interface PrimitiveFloatList extends List<Float> {
+  float getPrimitive(int index);
+  boolean addPrimitive(float o);
```

Comment:
worth documenting the boolean return value of add (its non trivial, at least for me)

Reply:
Sure, I've copied the JavaDoc from `List::get` and from `Collection::add`, respectively.